.eslintignore

```diff
@@ -4,3 +4,5 @@ coverage
 examples
 doc
 .eslintrc.js
+scripts/updateERC20.js
+webpack.config.js
```

Comment:
please add a new line after this last line

Reply:
done